app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -12,7 +12,7 @@ promo_content:
     - content/train-to-be-a-teacher/promos/find-your-course
     - content/train-to-be-a-teacher/promos/events-near-you
 navigation: 20.05
-navigation_description: Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+navigation_description: Find out how to get QTS if you have a degree or you’re studying for one. Discover how to choose your postgraduate teacher training course. 
```

Comment:
Is 'Discover' the right word here? Sounds a bit like we are revealing a secret?

Get advice? Or something more like that? Appreciate that characters are an issue!